docs/src/main/asciidoc/security-openid-connect-web-authentication.adoc

```diff
@@ -65,11 +65,11 @@ The OpenID Connect extension allows you to define the configuration using the `a
 ----
 quarkus.oidc.auth-server-url=http://localhost:8180/auth/realms/quarkus
 quarkus.oidc.client-id=frontend
-quarkus.oidc.client-type=web-app
+quarkus.oidc.application-type=web-app
```

Comment:
Was it a typo in the doc or is it related to some recent code changes?

Reply:
It was something we discussed with Pedro, `client-type` was in the PR only but `application-type` was what made it to the master